app/templates/landing-page.html

```diff
@@ -45,7 +45,7 @@ <h5 class="u-mb-no u-fs-xxs">PERIOD</h5>
     <div>
 
       <div class="header">
-        <h1 class="header__title">Monthly Business Survey - Retail Sales Index</h1>
+        <h1 class="header__title">{{ bar_title }}</h1>
```

Comment:
I don't think we should be re-using this here, that bar_title variable should only be used for the bar title.

We should actually add a new variable to the data objects passed to these pages